libiocage/cli/list.py

```diff
@@ -24,11 +24,15 @@
 """list module for the cli."""
 import click
 import texttable
+import typing
 
 import libiocage.lib.Host
 import libiocage.lib.Jails
+import libiocage.lib.JailFilter
 import libiocage.lib.Logger
 
+supported_output_formats = ['table', 'csv', 'list']
```

Comment:
Maybe now. That's super easy. But *csv* and *list* come with some nice optimizations - they stream their output. For JSON that would be useless :) But I will add it because it's a low hanging fruit!

Reply:
`yaml` is even lower hanging fruit! It should also buy us XML iirc

Reply:
no no no double no

Reply:
@igalic I can understand that you want XML so badly, but we can't just disable all other list options just for that! :trollface: @skarekrow would be very disappointed if we drop the table output!